### ISSUE TITLE
"increased cell diameter" vs. stubby

### ISSUE THREAD

**Issue by ValWood** [pombase/fypo#3338]
11 | increased cell diameter (FYPO:0006615) [single allele genotypes]

112 | stubby vegetative cell (FYPO:0000024) [single allele genotypes]

all " stubby vegetative cell" have increased cell diameter?

most increased  cell diameter annotations  should  probably be "stubby"

the only exception is 
"cells with increased diameter" with "normal cell length
(stubby are  short and fat, but volume is likely the same)

**Comment by ValWood** [comment]
FYPO:0006617 | viable elongated vegetative cell with increased cell diameter
-- | --

how does this differ from "swollen"  (i.e wider and longer?)

**Comment by JackyVH** [comment]
Yes they could be annotated to the stubby, though Kelly didn't measure the cell length, they certainly look stubby.  For FYPO0006617 the  'stubby'cells were incubated in HU to show that they were still polarised  i.e. elongated.  To me this is an elongated cell , the cell is not getting bigger in all dimensions it just elongates and its diameter (though wider) remains about the same as it elongates  so I wouldn't say it is swollen. Wouldn't swollen imply that both the diameter and length were increasing at the same time

**Comment by ValWood** [comment]
Yes I thought about this after.

We have defined "swollen" as

A cell morphology phenotype in which a cell has a larger volume than normal. In a swollen cell, both length and diameter are greater than normal, although the cell is only considered elongated if the length:width ratio is also greater than normal.


but  to me swollen is a bit different than just larger in each dimension.
 Swollen is larger with some constraining feature (like the capacity of the skin  or the cell wall to extend).

In these cases, if there is elongation there is also growth? don't you agree.


This one is complicated by the fact that the HU is used to force the cells to elongate  (which the WT would also do), so in this case  i agree with youthey are "longer' and "wider' but they aren't "swollen".

I don't know how we deal with this though- will need to wait for Midori to get back from hols.

**Comment by ValWood** [comment]
Interestingly, I'm just finishing checking Ken's session, the final bit of this paper says:


Rescue of rga4Δ polarity defects by gef1Δ in a scd1low background appeared to conflict with a previous report that rga4Δ gef1Δ double mutants were wider than either rga4Δ or gef1Δ single mutants (Kelly and Nurse, 2011). We therefore reinvestigated cell dimensions of rga4Δ and gef1Δ single and double mutants in a fully wild-type background (Fig. 7D,E). Consistent with an earlier characterization (Das et al., 2007), rga4Δ cells were wider than wild-type cells. However, we also found that additional deletion of gef1 restored rga4Δ cells to normal width. Our results in a wild-type (scd1+) background thus contradict previous work (Kelly and Nurse, 2011) and suggest that increased width of rga4Δ (scd1+) cells is a consequence of global Gef1 activity competing,
albeit with limited success, against relatively strong local Scd1 activity.

**Comment by ValWood** [comment]
so in this example  scd1 with wee 1 OX.
Normally wee1 would be elongated (because it  continues to grow it elongates)

when combined with scd1 which is wider it  still elongates, but not as much, but it gets wider.

I agree with Jacky that  this isn't swelling, its growth  accompanied by a  loss of shape (width control)

![elongated with increased diameter not swollen](https://user-images.githubusercontent.com/7359272/43095050-82177a62-8eac-11e8-929d-526d9a70c228.jpg)

**Comment by JackyVH** [comment]
I've just looked again at the cell width  in PMID21849474 where they used HU to elongate the wide mutants. According to Fig1  2 mutant deletes (its not clear which genes they are) the cells are longer and wider in HU as Ken saw with wee1 o/e and I suspect these two mutants are  scd1D and scd2D  as neither gets vey long, particularly scd1D, and both look wider in HU (Fig2).  However for the rest  of the wide mutants there is little  change in the width as the cells elongate  so the elongation is not always associated with a further increase in width. Not quite sure why I'm saying this except to agree with you that it is growth and not swelling but thought I would mention it. Also I can't explain why there is a difference between Felice Kelly and Ken's  results for rga4Dgef1D.

**Comment by ValWood** [comment]
;)

**Comment by mah11** [comment]
> all "stubby vegetative cell" have increased cell diameter?

Yes, I reopened #3326 to remind me to add a link

**Comment by mah11** [comment]
> FYPO:0006617  viable elongated vegetative cell with increased cell diameter
>
> how does this differ from "swollen" (i.e wider and longer?)
> [and follow-up comments]

I admit I'm not free of confusion now, and it's enirely possible that we should change something, whether or not the changes include any merges. At present:

We've defined swollen as increased volume, without bringing in any implications about either shape or growth pattern (or whatever else could lead to a cell being swollen). In other words, the def only says what a swollen cell looks like, not how it came to be swollen.

Whatever happens to other terms, I think I should improve the definition of "elongated cell" (FYPO:0000017 and subtypes), because it doesn't explicitly include what we say about elongated cells in the "swollen" defs - that to count as elongated, a cell has to have a greater length:diameter ratio than wild type. It's alluded to in a comment but ought to be clearer, and right in the def. Also, there isn't a separate term for just "increased length" but I think I'll add one.

If anything should merge with FYPO:0006617, it would be FYPO:0002479 ! viable swollen elongated vegetative cell. FYPO:0006617 explicitly says greater length and greater diameter, and that kind of implies a larger volume than normal. But it's also possible that instead of merging, we should make FYPO:0002497 is_a FYPO:0006617. It depends on what we decide about the fussy distinctions among swollen (inc. volume), increased length, elongated (increased length:diameter ratio):

swollen = increased volume (no other criteria)
elongated = increased length AND increased length:diameter ratio
FYPO:0006617 = increased length AND increased diameter, implying increased volume; does NOT say increased length:diameter ratio
swollen elongated = increased volume AND increased length AND increased length:diameter ratio

So at the moment, FYPO:0002479 is the most specific term of the above four, and no two are exactly identical. But I don't know whether all of the distinctions are worth making (biologically meaningful, useful for analysis, etc.).

IIRC, I went with what you (Val & Jacky) recommended to get the current definitions of swollen and elongated, but one or both can change if they don't still make sense. Let me know what works best ...

**Comment by mah11** [comment]
... and finally

> increased cell diameter (FYPO:0006615)
> currently is_a is_a elongated cell?

Due to an anomaly in PATO (diameter is_a length). I'll enquire, but (as you know) I can't tell how long it'll take to get it sorted. Ow my head.

**Comment by ValWood** [comment]
oh, dear, they are using length loosely as in 'any measurement'.Ow....

The subtle distinction between larger and swollen didn't cross my mind until the curation meeting...I don't know what we said originally. We can come back to this at the next curator meeting if we are still not there....

**Comment by mah11** [comment]
opened https://github.com/pato-ontology/pato/issues/172

**Comment by mah11** [comment]
I've done the bits that fall under "whatever happens to other terms" - edited the "elongated" defs, and added:

  increased cell length FYPO:0006633
  increased vegetative cell length FYPO:0006634

edit file: 47ad3a835a58e6c143b7c8f204006ebf4f8b7789
release: 81e41b8f7725fd2c5603f84cb1d9faa9007dcb91

**Comment by mah11** [comment]
> opened pato-ontology/pato#172

PATO fix for this is done in their edit file, so will come along whenever the next PATO release happens.

That just leaves the questions from [this comment](https://github.com/pombase/fypo/issues/3338#issuecomment-409623956), I think.
